cmd/build/main.go

```diff
@@ -64,8 +65,10 @@ func run() error {
 		return fmt.Errorf("invalid or unsupported distribution: %q", distroName)
 	}
 
-	archName := arch.Current().String()
-	arch, err := distribution.GetArch(archName)
+	if archName == "" {
+		archName = arch.Current().String()
+	}
+	archi, err := distribution.GetArch(archName)
```

Comment:
Nitpick: I don't have any problem with the rename of `arch` -> `archi`, just wanted to point it out in case this was not intentional.

Reply:
Its needed because we also import `arch` so if we have `arch` there the `arch` module is shadowed (and a few lines below iirc this code access the arch module now).